lib/tools/android-manifest.js

```diff
@@ -105,6 +105,17 @@ manifestMethods.targetSdkVersionFromManifest = async function (localApk) {
   }
 };
 
+manifestMethods.targetSdkVersionUsingPKG = async function (pkg) {
+  let stdout =  await this.shell(['dumpsys', 'package', pkg]);
+  let targetSdkVersion = new RegExp(/targetSdk=([^\s\s]+)/g).exec(stdout);
+  if (targetSdkVersion && targetSdkVersion.length >= 2) {
+    targetSdkVersion = targetSdkVersion[1];
+  } else {
+    targetSdkVersion = null;
+  }
+  return parseInt(targetSdkVersion, 10);
```

Comment:
`parseInt(null, 10)` returns `NaN` which is probably not what we mean. I'd either return `null` explicitly or return 0.